source/_integrations/androidtv.markdown

```diff
@@ -76,6 +76,11 @@ get_sources:
   required: false
   default: true
   type: boolean
+source_provider:
+  description: Whether to populate the list of sources from running_apps or installed_apps
```

Comment:
I don't know if there's a way to make it so that it shows the possible options, but you should specify that the choices are `running_apps` and `installed_apps`.  As it is, it's a bit ambiguous.